src/qz/printer/status/NativeStatus.java

```diff
@@ -95,13 +80,17 @@
         return fromWmi(bitwiseCode, printer, nativeType, -1, null);
     }
 
-    static Status fromCups(String reason, String printer, NativeType nativeType) {
-        if (reason == null) { return null; }
 
+    static Status fromCups(String reason, String state, String printer, int jobId, NativeType nativeType) {
```

Comment:
There's no point in passing in `nativeType` if you're going to assume Job/Printer based on function signature.  I also feel this diverges it from Wmi, we should probably settle on something that makes them similar.